src/Commands/BranchDeployVercelCommand.php

```diff
@@ -19,6 +19,9 @@ class BranchDeployVercelCommand extends Command {
 
     public InputInterface $input;
     public OutputInterface $output;
+    public $BACKEND_KEY = 'NEXT_PUBLIC_BACKEND_URL';
```

Comment:
Is it going to have the literal `NEXT_PUBLIC_BACKEND_URL` ?

Reply:
no it's the key, not the value